### PR TITLE
Turns lightgeists into a basic mob

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request

https://www.youtube.com/watch?v=bcUuqKrnvVY <- me getting rid of every simple animal one thing at a time

Turns lightgeists into a basic mob. There are a few key advantages into its implementation that was previously impossible in the simple animal framework, which is entirely possible now.
## Why It's Good For The Game

Lightgeists can actually have a bit of AI now, as a treat. They didn't have it prior because I assume if they attacked anything, they would end up healing it, and we didn't really have an option in the simple animal AI to say "do not attack"... This really fixes that so they can wander around a bit and look cute if some admin decides to spawn them in (but the only way to get them outright is to use the anomalous crystal, which auto-assigns a player). You don't see the AI if a player ghosts out of the lightgeist because the lightgeist will delete itself on ghost (previous behavior).

It also gets rid of some janky AttackingTarget() override that was completely un-needed post-#73202.
## Changelog
Nothing really player facing.

### CODE REVIEW

File: code/modules/mob/living/basic/space_fauna/lightgeist.dm [hunk 1]
```diff
@@ -57,19 +65,15 @@
...
 	. = ..()
 	if(.)
 		death()
 
-/mob/living/simple_animal/hostile/lightgeist/AttackingTarget()
-	if(istype(target, /obj/structure/ladder)) //special case where lightgeists can use ladders properly.
-		var/obj/structure/ladder/laddy = target
-		laddy.use(src)
+/// This is a bit neutered since these aren't intended to exist outside of player control, but it's a bit weird to just have these guys be completely stationary.
+/// No attacking or anything like that, though. Just something so they seem alive.
```
Comment (LemonInTheDark) [inline code/modules/mob/living/basic/space_fauna/lightgeist.dm:75]: They're used in the clog event tho, right? don't they need ai behavior?

Comment (san7890) [reply code/modules/mob/living/basic/space_fauna/lightgeist.dm:75]: image


the old implementation had them do absolutely nothing except just stand there. this is an improvement

File: code/modules/mob/living/basic/space_fauna/lightgeist.dm [hunk 2]
```diff
@@ -35,17 +36,24 @@
...
+	ai_controller = /datum/ai_controller/basic_controller/lightgeist
+
+/mob/living/basic/lightgeist/Initialize(mapload)
 	. = ..()
+	ADD_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+
+	remove_verb(src, /mob/living/verb/pulled)
+	remove_verb(src, /mob/verb/me_verb)
+
+	var/datum/atom_hud/medsensor = GLOB.huds[DATA_HUD_MEDICAL_ADVANCED]
```
Comment (Mothblocks) [inline code/modules/mob/living/basic/space_fauna/lightgeist.dm:54]: `medical_sensor` please

**Review by Mothblocks — APPROVED** [review]

**Comment by Jacquerel** [comment]
I was honestly considering giving them AI where they _would_ attack injured nearby entities (to heal them) but I can do that whenever.

**Pull request merged**